charts/minecraft-proxy/templates/deployment.yaml

```diff
@@ -151,6 +150,8 @@ spec:
         {{- end }}
         {{- end }}
         volumeMounts:
+        - name: tmp
```

Comment:
I wonder if this is actually necessary for the proxy?

Reply:
It seems best to include since

https://github.com/itzg/docker-bungeecord/blob/608f06bc983a3b5084e54ec299477d18089ab8e7/run-bungeecord.sh#L152